.github/workflows/commit.yml

```diff
@@ -264,3 +222,38 @@ jobs:
           COSIGN_PASSWORD: ${{ secrets.COSIGN_PASSWORD }}
           TOOL: ${{ matrix.tool }}
           ENVIRONMENT: ${{ matrix.environment }}
+  finalize-the-release:
+    name: Finalize the release
+    needs: [bump-version, distribute]
+    if: "${{ github.event_name == 'push' && !startsWith(github.event.head_commit.message, 'Bump version: 2') }}"
```

Comment:
This is me being overly cautious. The `needs` section depending on `bump-version` or `distribute` should be enough, but why not.